internal/cmd/db_transfer.go

```diff
@@ -0,0 +1,70 @@
+package cmd
+
+import (
+	"fmt"
+
+	"github.com/chiselstrike/iku-turso-cli/internal"
+	"github.com/chiselstrike/iku-turso-cli/internal/prompt"
+	"github.com/chiselstrike/iku-turso-cli/internal/settings"
+	"github.com/chiselstrike/iku-turso-cli/internal/turso"
+	"github.com/spf13/cobra"
+)
+
+func init() {
+	dbCmd.AddCommand(dbTransferCmd)
+}
+
+var dbTransferCmd = &cobra.Command{
+	Use:               "transfer database_name org_name",
+	Short:             "Transfers database to another organization",
+	Args:              cobra.ExactArgs(2),
+	ValidArgsFunction: dbNameArg,
```

Comment:
This will make autocomplete suggest DB name for both arguments.
If you're going to add a `ValidArgsFunction` you'll probably need to call `dbNameArg` for the first argument and something similar for orgs on the second argument.

Reply:
fixed